debian/control

```diff
@@ -62,7 +69,7 @@ Package: golang-doc
 Replaces: golang-weekly-doc
 Conflicts: golang-weekly-doc
 Provides: golang-weekly-doc
-Depends: ${misc:Depends}, golang-go
+Depends: golang-go, ${misc:Depends}
```

Comment:
This was the result of `wrap-and-sort`, so I'm not opposed to taking it out for a smaller delta for 1.4->1.5, and will update. :+1: